### PR TITLE
[CL-396] Update theme colors to new hexes

### PR DESCRIPTION
**Pull request by vleague2**
## 🎟️ Tracking

[CL-396](https://bitwarden.atlassian.net/browse/CL-396)

## 📔 Objective

This PR updates the base color palette as per the most recent colors from design.

## 📸 Screenshots

Check Storybook for changes (recommend the Colors page and the Kitchen Sink).

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[CL-396]: https://bitwarden.atlassian.net/browse/CL-396?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/components/src/tw-theme.css [hunk 1]
```diff
@@ -9,36 +9,39 @@
   --color-background-alt3: 18 82 163;
   --color-background-alt4: 13 60 119;
```
Comment (danielleflinn) [inline libs/components/src/tw-theme.css:11]: Not sure if I missed this or if we missed it in the last color update but in Figma I have the following for background-alt:
`background-alt=243, 246, 249`

File: libs/components/src/tw-theme.css [hunk 2]
```diff
@@ -62,39 +65,42 @@
...
   --color-danger-700: 255 133 148;
 
-  --color-warning-100: 87 52 0;
-  --color-warning-600: 255 187 87;
-  --color-warning-700: 255 207 138;
+  --color-warning-100: 172 88 0;
+  --color-warning-600: 255 191 0;
+  --color-warning-700: 255 248 228;
+
+  --color-info-100: 26 66 172;
```
Comment (danielleflinn) [inline libs/components/src/tw-theme.css:92]: `info-100= 26, 65, 172`

File: libs/components/src/tw-theme.css [hunk 3]
```diff
@@ -62,39 +65,42 @@
...
-  --color-danger-100: 82 0 10;
-  --color-danger-600: 255 82 103;
+  --color-danger-100: 149 27 42;
+  --color-danger-600: 255 78 99;
   --color-danger-700: 255 133 148;
 
-  --color-warning-100: 87 52 0;
-  --color-warning-600: 255 187 87;
-  --color-warning-700: 255 207 138;
+  --color-warning-100: 172 88 0;
```
Comment (danielleflinn) [inline libs/components/src/tw-theme.css:88]: `danger-700=255, 236, 239`

**Review by danielleflinn — CHANGES_REQUESTED** [review]
Overall looks great! I caught a few typos which I've commented on

File: libs/components/src/tw-theme.css (hunk 3 above)
Comment (vleague2) [reply libs/components/src/tw-theme.css:88]: Wow I wasn't even close.....

Comment (vleague2) [reply libs/components/src/tw-theme.css:88]: Wait, danger or warning?

Comment (danielleflinn) [reply libs/components/src/tw-theme.css:88]: Dark theme danger; idk how the comment ended up on warning 😬 You hadn't changed danger-700 at all so maybe Github cropped out the line?

Comment (merissaacosta) [reply libs/components/src/tw-theme.css:88]: just to clarify `danger` -> `warning` and `warning` -> `info` right?

Comment (vleague2) [reply libs/components/src/tw-theme.css:88]: No, the variable names aren't changing, she just commented on the wrong line and we were confused 😂

**Review by danielleflinn — APPROVED** [review]
Storybook looks good!

**Pull request merged**